woocommerce/woocommerce-ios
# Blaze Optimization: Add new entity BlazeCampaign to Core Data model

Part of #10907 


## Description

This PR adds a new entity `BlazeCampaign` to Core Data to support persisting campaigns locally.

## Testing instructions

Build the app to ensure that migration works successfully (no crash). CI should pass too.

## Screenshots

N/A

---
